package.json

```diff
@@ -20,7 +20,7 @@
     "joi": "^17.7.0",
     "morgan": "^1.10.0",
     "node-cache": "^5.1.2",
-    "openai": "~4.24.1",
+    "openai": "4.40.0",
```

Comment:
We can surely but my only worry is if open AI repeats the same mistake that  they did recently and our signal system failed it may happen here as well? So need your suggestion on that

Reply:
Basically 4.40.0 is the most stable version , and this same version is being used by signal api